AGENTS.md

```
@@ -134,16 +134,24 @@
-
-# 2. Add test files (main.py + any supporting modules)
+# Add test files (main.py + any supporting modules)
 echo "print('Hello Feature')" > crates/cribo/tests/fixtures/my_new_feature/main.py
 
-# 3. Run tests - automatically discovered and tested
-cargo test test_all_bundling_fixtures
+# Run a specific fixture using environment variable
+INSTA_GLOB_FILTER="**/my_new_feature/main.py" cargo nextest run --test test_bundling_snapshots --cargo-quiet --cargo-quiet
+
+# Run all fixtures matching a pattern
+INSTA_GLOB_FILTER="**/future_imports_*/main.py" cargo nextest run --test test_bundling_snapshots --cargo-quiet --cargo-quiet
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)